vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -84,13 +84,9 @@
     <Import Project="$(SolutionDir)packages\boost.1.68.0.0\build\boost.targets" Condition="Exists('$(SolutionDir)packages\boost.1.68.0.0\build\boost.targets')" />
   </ImportGroup>
   <ItemGroup>
-    <ClInclude Include="$(JSI_Source)\jsi\decorator.h" />
-    <ClInclude Include="$(JSI_Source)\jsi\instrumentation.h" />
-    <ClInclude Include="$(JSI_Source)\jsi\jsi-inl.h" />
-    <ClInclude Include="$(JSI_Source)\jsi\jsi.h" />
-    <ClInclude Include="$(JSI_Source)\jsi\JSIDynamic.h" />
```

Comment:
>\JSIDynamic.h" [](start = 41, length = 14)

Why these files are removed from project?

Reply:
They are unused (and cause even further issues with our triplicate copies of JSI)

---
In reply to: [387943756](https://github.com/microsoft/react-native-windows/pull/4234#discussion_r387943756) [](ancestors = 387943756)